flex/views/experiment/TabSampleSetupView.mxml

```diff
@@ -232,7 +232,7 @@
 			}
 			private function propagateSeqLibProtocol():void {
 				for each (var sample:Object in parentDocument.samples) {
-					if (currentState == 'SolexaSetupPreppedState' || currentState == 'SolexaAmendPreppedState' || (seqPrepRadioGroup.selectedValue !=  null && seqPrepRadioGroup.selectedValue == 'N') ) {
+					if (currentState == 'SolexaSetupPreppedState' || currentState == 'SolexaAmendPreppedState' ) {

```

Comment:
Actually here the method propagateSeqLibProtocol isn't even changing sample.@seqLibProtocol at all - it's changing sample.@seqPrepByCore instead. The mistake has been here a while, though, independent of your pull request.